src/app/public/modules/format/format.ts

```diff
@@ -3,6 +3,9 @@ export {
   SkyAppFormat
 } from './app-format';
 
+/**
+ * @deprecated Please use `SkyAppFormat` instead.
```

Comment:
@blackbaud-johnly This message and the one below need your approval, please.

Reply:
I'd change this one to just 
```Use `SkyAppFormat` instead.```